ts/src/bingx.ts

```diff
@@ -2753,6 +2755,48 @@ export default class bingx extends Exchange {
         return response;
     }
 
+    async cancelAllOrdersAfter (timeout: Int, activated: Bool = undefined, params = {}) {
+        /**
+         * @method
+         * @name bingx#cancelAllOrdersAfter
+         * @description dead man's switch, cancel all orders after the given timeout
+         * @see https://bingx-api.github.io/docs/#/en-us/spot/trade-api.html#Cancel%20all%20orders%20in%20countdown
+         * @see https://bingx-api.github.io/docs/#/en-us/swapV2/trade-api.html#Cancel%20all%20orders%20in%20countdown
+         * @param {number} countdown time in seconds
```

Comment:
@sc0Vu the default unit is milliseconds we should never accept seconds as a unified parameter